portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+  
+  private String name;
+  private String content;
```

Comment:
These can be final

Reply:
timestamp could be as well, correct? since it's assigned once and never altered

Reply:
Correct :)